### PR TITLE
feat(batch): update batch method

### PR DESCRIPTION
**Pull request by enddynayn**
Update batch method to match solidity contract interface.

### CODE REVIEW

File: src/contracts/announcement.test.ts [hunk 1]
```diff
@@ -20,15 +20,20 @@ describe("#batch", () => {
...
+      },
+    ];
 
     await setConfig({
       accountAddress: account.address as EthereumAddress,
       provider: web3,
     } as Config);
 
-    const receipt = await batch(testUri, hash);
+    const receipt = await batch(annoucements);
```
Comment (shannonwells) [inline src/contracts/announcement.test.ts:36]: typo:
```suggestion
    const receipt = await batch(announcements);
```

File: src/contracts/announcement.test.ts [hunk 2]
```diff
@@ -20,15 +20,20 @@ describe("#batch", () => {
   it("successfully posts a batch to the chain", async () => {
     jest.setTimeout(12000);
 
-    const testUri = "http://www.testconst.com";
-    const hash = keccak256("test");
+    const annoucements: Announcement[] = [
```
Comment (shannonwells) [inline src/contracts/announcement.test.ts:23]: typo:
```suggestion
    const announcements: Announcement[] = [
```

**Review by shannonwells — COMMENTED** [review]
Doesn't Announcer.d.ts need to be updated too?

**Review by shannonwells — APPROVED** [review]

**Review by wilwade — COMMENTED** [review]
Tagging for a rebase and updating the contracts dependency to `0.0.0-83c461`

**Review by aneyzberg — APPROVED** [review]
lgtm once rebased and tests pass

**Pull request merged**